code/modules/mob/language.dm

```diff
@@ -411,33 +439,42 @@
 	new_name += " [pick(list("Alkaid","Andromeda","Antlia","Apus","Auriga","Caelum","Camelopardalis","Canes Venatici","Carinae","Cassiopeia","Centauri","Circinus","Cygnus","Dorado","Draco","Eridanus","Errakis","Fornax","Gliese","Grus","Horologium","Hydri","Lacerta","Leo Minor","Lupus","Lynx","Maffei","Megrez","Messier","Microscopium","Monocerotis","Muscae","Ophiuchi","Orion","Pegasi","Persei","Perseus","Polaris","Pyxis","Sculptor","Syrma","Telescopium","Tianyi","Triangulum","Trifid","Tucana","Tycho","Vir","Volans","Zavyava"))]"
 	return new_name
 
-/datum/language/common
+
+///Human misc langauges, able to be learnt by every species, however are some origin from Humans
+/datum/language/common //This is the default language everyone should obviously start with, so its always going to be given to crew members unless something something admins
 	name = "Galactic Common"
-	desc = "The common galactic tongue."
+	desc = "A fusion of Human and Skrellian dialects, it stands as one of the most universally comprehended languages throughout the Orion Sector. It has gained immense popularity, particularly within the Trans-Solar Federation, where it serves as a common tongue understood by virtually all its inhabitants, whether they hail from the planets or the stars. Proficiency in Galactic Common has become almost mandatory for securing employment or prospects within Human or Skrell territory. Its widespread usage and long history have contributed to making Galactic Common one of the easiest and most straightforward languages to learn."
```

Comment:
Just for my own sanity, can you see this text anywhere in-game or is it just code only